src/snabbdom.bundle.ts

```diff
@@ -1,16 +0,0 @@
-import {init} from './snabbdom';
```

Comment:
dropped entirely. in v2 we only give you the low-level stuff, something like this can be a different package with some opinionated diffing configuration.